scripts/yum/plugins.py

```diff
@@ -0,0 +1,781 @@
+#! /usr/bin/python -tt
```

Comment:
Is this file being moved from somewhere? If files get moved then xapi.spec might need to be updated too.

Reply:
It's a copy to get some of the checking working, to be fixed properly in that sub-PR later.